pay-api/tests/unit/services/test_statement.py

```diff
@@ -172,7 +172,7 @@ def test_get_weekly_interim_statement(session, admin_users_mock):
                                      total=50).save()
 
     assert weekly_invoice is not None
-    assert weekly_invoice.created_on == invoice_create_date.astimezone(timezone.utc).replace(tzinfo=None)
+    assert localize_date(weekly_invoice.created_on) == invoice_create_date
```

Comment:
This works locally, not sure if it will work for UTC though - I probably should put the database into UTC mode, similar to LEAR:

`    sess.execute(text("SET TIME ZONE 'UTC';"))`